packages/wdio-dot-reporter/README.md

```diff
@@ -25,6 +25,8 @@ npm install @wdio/dot-reporter --save-dev
 
 Instructions on how to install `WebdriverIO` can be found [here](/docs/gettingstarted.html).
 
+**V5 Note:**  Dot reporter does not support test failure output in version 5.  You can track progress on this feature [here](https://github.com/webdriverio/webdriverio/pull/3589).  [Spec Reporter](/docs/spec-reporter.html) can be used in the meantime.
```

Comment:
No need to make a special v5 note. The current docs that you find on https://webdriver.io/ are for v5 only the other docs are in http://v4.webdriver.io/